contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2409,13 +2409,13 @@ GRANT SELECT ON sys.registered_search_property_lists TO PUBLIC;
 CREATE OR REPLACE VIEW sys.filegroups
 AS
 SELECT 
-   ds.name,
-   ds.data_space_id,
-   ds.type,
-   ds.type_desc,
-   ds.is_default,
-   ds.is_system,
-   CAST(NULL as UNIQUEIDENTIFIER) AS filegroup_guid,
+   CAST(ds.name AS sys.SYSNAME),
+   CAST(ds.data_space_id AS INT),
+   CAST(ds.type AS sys.BPCHAR(2)),
```

Comment:
shall we use char(2) instead? BPCHAR is an internal type

Reply:
same as below

Reply:
@Deepesh125 Advised otherwise, that I use `sys.bpchar` instead of `char`. Here is a transcript of the conversation around this topic in a previous code review:

**Dipesh:** use sys.char(2)

**Me:** When I tried to use sys.char(2) as such -- , `CAST(obj.type as sys.char(2)) AS type`, BBF failed to initialize with the following error: `ERROR:  type "sys.char" does not exist`. Is this expected?

**Dipesh:** `sys.varchar` and `sys.nvarchar` works fine and sys.char does not? There must be something wrong? can you do some debugging?

**Me:** I did some digging and found we are missing this type of statement in the code: `CREATE DOMAIN sys.CHAR AS sys.BPCHAR;`

I added this statement to `RDSManfredBabelExtensions/contrib/babelfishpg_common/sql/bpchar.sql` & then was able to use `sys.char`. I'm not 100% sure if `sys.char` is supposed to be mapped to `sys.bpchar`, but anyway it seems the main issue was this missing this CREATE statement.

Let me know if I'm on the right track/next steps I should take to get this approved.

**Dipesh:** Okay.. I've filed BABEL-3368 and let's use `sys.bpchar` for the time being.